internal/test/outputoptions/name-normalizer/to-camel-case-with-additional-initialisms/name_normalizer_test.go

```diff
@@ -0,0 +1,24 @@
+package tocamelcasewithadditionalinitialisms
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenCodeHasCorrectNames(t *testing.T) {
+	pet := &Pet{}
+	assert.Empty(t, pet.NAME)
+	assert.Empty(t, pet.UUID)
+
+	uri := "https://my-api.com/some-base-url/v1/"
+	client, err := NewClient(uri)
+	assert.Nil(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)
```

Reply:
For record, I copy/paste exactly for the other tests ...

Reply:
This repository should enable testifylint via golangci-lint.

And yes, previous test could have been better